config/boot.rb

```diff
@@ -1,3 +1,4 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+#require 'bootsnap/setup'
```

Comment:
How come it's commented out?

Reply:
> How come it's commented out?

Ha, good spot. That's me just testing how the performance changed without bootsnap. I've removed the `#` now.